packages/uniwind/tests/test.css

```
@@ -40,3 +40,8 @@
   flex-shrink: 1;
   flex-basis: 0%;
 }
+
+@utility var-cascade-after {
```

Stylelint (17.11.1)

[error] Unexpected unknown at-rule "`@utility`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)